crates/pipeline_manager/src/main.rs

```diff
@@ -388,85 +443,185 @@ where
-                    .body(format!("invalid program id '{program_id}': {e}"))),
-                Ok(program_id) => Ok(ProgramId(program_id)),
-            }
-        }
-    }
+// Example errors for use in OpenApi docs.
+
+fn example_unknown_program() -> ErrorResponse {
+    ErrorResponse::from_error(&DBError::UnknownProgram {
+        program_id: ProgramId(uuid!("67e55044-10b1-426f-9247-bb680e5fe0c8")),
+    })
+}
+
+fn example_duplicate_name() -> ErrorResponse {
+    ErrorResponse::from_error(&DBError::DuplicateName)
+}
+
+fn example_outdated_program_version() -> ErrorResponse {
+    ErrorResponse::from_error(&DBError::OutdatedProgramVersion {
+        expected_version: Version(5),
+    })
+}
+
+fn example_unknown_pipeline() -> ErrorResponse {
+    ErrorResponse::from_error(&DBError::UnknownPipeline {
+        pipeline_id: PipelineId(uuid!("2e79afe1-ff4d-44d3-af5f-9397de7746c0")),
+    })
+}
+
+fn example_unknown_connector() -> ErrorResponse {
+    ErrorResponse::from_error(&DBError::UnknownConnector {
+        connector_id: ConnectorId(uuid!("d764b9e2-19f2-4572-ba20-8b42641b07c4")),
+    })
+}
+
+fn example_unknown_name() -> ErrorResponse {
+    ErrorResponse::from_error(&DBError::UnknownName {
+        name: "unknown_name".to_string(),
+    })
+}
+
+fn example_unknown_input_table(table: &str) -> ErrorResponse {
+    ErrorResponse::from_error(&ControllerError::unknown_input_stream(table))
+}
+
+fn example_unknown_output_table(table: &str) -> ErrorResponse {
+    ErrorResponse::from_error(&ControllerError::unknown_output_stream(table))
+}
+
+fn example_unknown_input_format() -> ErrorResponse {
+    ErrorResponse::from_error(&ControllerError::unknown_input_format("xml"))
+}
+
+fn example_parse_error() -> ErrorResponse {
+    ErrorResponse::from_error(&ControllerError::parse_error(
+        "api-ingress-my_table-d24e60a3-9058-4751-aa6b-b88f4ddfd7bd",
+        &"missing field 'column_name'",
+    ))
+}
+
+fn example_unknown_output_format() -> ErrorResponse {
+    ErrorResponse::from_error(&ControllerError::unknown_output_format("xml"))
+}
+
+fn example_pipeline_shutdown() -> ErrorResponse {
+    ErrorResponse::from_error(&RunnerError::PipelineShutdown {
+        pipeline_id: PipelineId(uuid!("2e79afe1-ff4d-44d3-af5f-9397de7746c0")),
+    })
+}
+
+fn example_program_not_set() -> ErrorResponse {
+    ErrorResponse::from_error(&RunnerError::ProgramNotSet {
+        pipeline_id: PipelineId(uuid!("2e79afe1-ff4d-44d3-af5f-9397de7746c0")),
+    })
+}
+
+fn example_program_not_compiled() -> ErrorResponse {
+    ErrorResponse::from_error(&RunnerError::ProgramNotCompiled {
+        pipeline_id: PipelineId(uuid!("2e79afe1-ff4d-44d3-af5f-9397de7746c0")),
+    })
+}
+
+fn example_pipeline_timeout() -> ErrorResponse {
+    ErrorResponse::from_error(&RunnerError::PipelineInitializationTimeout {
+        pipeline_id: PipelineId(uuid!("2e79afe1-ff4d-44d3-af5f-9397de7746c0")),
+        timeout: STARTUP_TIMEOUT,
```

Comment:
STARTUP_TIMEOUT is imported from runner. Is this the only place it is used?

Reply:
yes, and also inside `runner.rs` itself.